contracts/test/Tracker.t.sol

```diff
@@ -58,6 +60,44 @@ contract TrackerTest is Test {
         tracker.settle(0);
         (,,,,,,, bool settled) = tracker.challenges(0);
         require(settled, "Settle failed");
-        require(tracker.balances(address(this)) == 0.0001 ether, "Settle failed");
+        require(tracker.getClaimableAmount(0, address(this)) == 0.0001 ether, "Settle failed");
+    }
+
+    function test_withdraw(uint256 timestamp) public {
+        vm.assume(timestamp > 0 && timestamp < 1e20);
+        commonJoinAndCheckIn(timestamp);
+
+        vm.warp(block.timestamp + 1001);
+        tracker.settle(0);
+        uint256 beforeBalance = underlying.balanceOf(address(this));
+        tracker.withdraw(0);
+        uint256 afterBalance = underlying.balanceOf(address(this));
+        require(afterBalance - beforeBalance == PER_USER_STAKE, "withdraw failed");
+    }
+
+    function test_withdraw_failed_user(address failedUser, uint256 timestamp) public {
```

Comment:
better use some prefix like `test_RevertWhen` to indicate that this test is expected to revert.
This way people can use flags like `--match-test` to exclude these tests when running gas reports.

Ref: https://github.com/sablier-labs/v2-core/blob/9657fbd0a77821a088b34d460dd1117e5600bb43/test/unit/concrete/adminable/transfer-admin/transferAdmin.t.sol#L9